device.h

```diff
@@ -71,6 +72,10 @@ struct vcam_device {
     char vcam_fb_fname[FB_NAME_MAXLENGTH];
     struct proc_dir_entry *vcam_fb_procf;
     struct mutex vcam_mutex;
+    /* framebuffer info  */
+    struct fb_info info;
```

Comment:
The name `info` might be too generic to distinguish. Clarify it as well.

Reply:
Alternatively, you can replace the three members into `void *fb_priv`, which encapsulates framebuffer specific data. And, you don't have to include `<linux/fb.h>` in header `device.h`.